src/node/utils/caching_middleware.js

```diff
@@ -196,7 +196,7 @@ CachingMiddleware.prototype = new function () {
         var statusCode = responseCache[cacheKey].statusCode;
 
         var pathStr = CACHE_DIR + 'minified_' + cacheKey;
-        if (supportsGzip && (headers['content-type'] || '').match(/^text\//)) {
+        if (supportsGzip && (headers['content-type'] || '').match(/^application\/javascript/)) {
```

Comment:
That shouldn't be necessary, because the code path is only taken when requesting packages or javascript files from plugins (paths that start with `/javascripts/`). Am I missing something?